### DISCUSSION TITLE
title en button names appears wrong

### DISCUSSION THREAD

**Discussion by plivive** [sorayuki/obs-multi-rtmp#261] [General]
Hello guys,
I am running OBS 29.0.2 and I installed obs-multi-rtmp 0.2.8.2
for some reason the names of the buttons and title appears wrong.
see attached
any clues about what could be the problem?

Thx.
![Schermafbeelding 2023-03-28 022857](https://user-images.githubusercontent.com/13613838/228096149-2487fd92-4758-4249-a827-4668c11ae1e8.png)

**Comment by sorayuki** [comment]
What your language is?
________________________________
From: plivive ***@***.***>
Sent: Tuesday, March 28, 2023 8:31:10 AM
To: sorayuki/obs-multi-rtmp ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [sorayuki/obs-multi-rtmp] title en button names appears wrong (Discussion #261)


Hello guys,
I am running OBS 29.0.2 and I installed obs-multi-rtmp 0.2.8.2
for some reason the names of the buttons and title appears wrong.
see attached
any clues about what could be the problem?

Thx.
[Schermafbeelding 2023-03-28 022857]<https://user-images.githubusercontent.com/13613838/228096149-2487fd92-4758-4249-a827-4668c11ae1e8.png>

―

**Reply by plivive** [reply]
My OBS is English language
My Windows10 is Dutch language

**Comment by plivive** [comment]
My OBS is English language 
My Windows10 is Dutch language 

![screen](https://user-images.githubusercontent.com/13613838/228158110-c9e7ac50-5177-4574-bbc8-5781c5274f73.jpg)
